server/app/views/applicant/ProgramIndexTemplate.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+  <head th:replace="~{applicant/ApplicantBaseFragment :: pageHeader}"></head>
+  <body>
+    <h1 th:text="#{content.findPrograms}"></h1>
+    <div th:text="#{content.findProgramsDescription}"></div>
```

Comment:
findProgramsDescription requires an argument. I think you'll need to pass through getWhitelabelCivicEntityShortName

Reply:
Intentionally left that out of this change, is coming in the next one